src/Console/Commands/SyncCommand.php

```diff
@@ -51,7 +51,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $ignoreColumns = $input->getOption('ignore-column');
         $noData= $input->getOption('no-data') ? true : false;  
         $unbuffered =  $input->getOption('un-buffer') ? true : false;  
-        
+        $keepUnbuffer = $input->getOption('keep-unbuffer') ? true : false;
```

Comment:
Note that by default if the option is not provided it will be false, if it is provided it will be true.

InputOption::VALUE_NONE
Do not accept input for this option (e.g. --yell). The value returned from is a boolean (false if the option is not provided). This is the default behavior of options;

https://symfony.com/doc/current/console/input.html

Reply:
"Note that by default if the option is not provided it will be false, if it is provided it will be true." <- yeah that sounds  up branch they are deleting them by default